plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/guru/VMwareGuru.java

```diff
@@ -1146,6 +1146,7 @@ public List<Command> finalizeMigrate(VirtualMachine vm, Map<Volume, StoragePool>
     private String getTargetHostGuid(StoragePool targetLocalPoolForVM, Long destClusterId, boolean isInterClusterMigration) {
         String targetHostGuid = null;
         if (targetLocalPoolForVM != null) {
+            // Get the target host for local storage migration
```

Comment:
:grin: this comment says kind of exactly what the code line below says as well. not sure we need it, but hey ...